### ISSUE TITLE
Transformations documentation: separate between preprocessing and data augmentation

### ISSUE THREAD

**Issue by uzaymacar** [ivadomed/ivadomed#1059]
As seen in the below lines

https://github.com/ivadomed/ivadomed/blob/76b36a0a0f7141feb2d5b00b33e4c3a06865fc2c/ivadomed/transforms.py#L1181

we specify (i) resampling, (ii) center-cropping, and (iii) roi-cropping as preprocessing steps amongst all the transformations.

As per the discussion today, we should probably make this separation clear in the documentation. We can have a "preprocessing" and "data augmentations" subsection under [all the transformations listed](https://ivadomed.org/configuration_file.html#transformations).

**Comment by jcohenadad** [comment]
Alternatively, we could have a heading 1 section "Data Augmentation" and another heading 1 section "Preprocessing". It would be coherent with the heading 1 "Postprocessing" that already exists.

**Comment by uzaymacar** [comment]
Agreed, that's better! We also need a separate heading for "other" utility transformations such as `NumpyToTensor()` and `NormalizeInstance()` as they are neither preprocessing nor data augmentation. I hope diffusing "Transformations" into the three separate headings discussed here wouldn't be too visually overwhelming.
